app/controllers/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+class UsersController < ApplicationController
+    get '/users/new' do
+        @user = User.new
```

Comment:
Using instance variables (`@user`) is a common enough pattern in Rails, but I lean away from using "magic" that leaks object scope and pass the variables in as locals instead.